canvas.go

```diff
@@ -248,6 +237,20 @@ func (c *Canvas) servicesGroup(canvas *svg.SVG) {
 	}
 }
 
+func (c *Canvas) iconClipPath(canvas *svg.SVG) {
```

Comment:
This method should have its own tests to ensure that we know the ramifications of changing serviceBlockSize or iconSize